### ISSUE TITLE
Could not find "lib" in source paths, while generating new app

### ISSUE THREAD

**Issue by roblingle** [rugui/rugui#23]
Trying to create a new app, I get the following output:

```
rob@ubuntu:~$ rugui hellorugui
  create  
  create  README
  create  Rakefile
  create  bin
  create  bin/hellorugui
   chmod  bin/hellorugui
  create  bin/hellorugui.bat
   chmod  bin/hellorugui.bat
  create  app
  create  app/main.rb
  create  config
  create  config/environments/test.rb.sample
  create  config/environments/production.rb.sample
  create  config/environments/development.rb.sample
  create  config/boot.rb
Could not find "lib" in source paths.
```

'rake run' in the app directory returns:

```
rob@ubuntu:~/hellorugui$ rake run
(in /home/rob/hellorugui)
rake aborted!
no such file to load -- /home/rob/hellorugui/app/../config/environment

(See full trace by running task with --trace)
```

The lib directory and the environment.rb are not created by the generator. I'll dig in a little more, but thought I'd mention in case it's an easy fix for someone already familiar.

**Comment by thecatwasnot** [comment]
FWIW I'm seeing the exact same problem.  I tried both the gem and creating a gem from the latest source and no dice.

**Comment by rugui** [comment]
Hi, I've just merged your changes into master and released 1.4.1. Check if it works now.

Cheers
